plugins/plugin-chart-echarts/src/BoxPlot/transformProps.ts

```diff
@@ -105,7 +105,7 @@ export default function transformProps(chartProps: ChartProps): EchartsProps {
   else if (xTicksLayout === '90°') axisLabel = { rotate: -90 };
   else if (xTicksLayout === 'flat') axisLabel = { rotate: 0 };
   else if (xTicksLayout === 'staggered') axisLabel = { rotate: -45 };
```

Comment:
Since we have a `xTicksLayout` control already? Why not just set the default of `xTicksLayout` to `rotate: -45`?